Doc/library/shlex.rst

```diff
@@ -43,15 +43,16 @@ The :mod:`shlex` module defines the following functions:
    string that can safely be used as one token in a shell command line, for
    cases where you cannot use a list.
 
-   This idiom would be unsafe::
+   This idiom would be unsafe:
```

Comment:
Mmm, I think there is a misunderstanding here.  I do not want to skip this test group, but I want it to be executed.  If we skip these tests then `remote_command` will not be defined and the next test groups (after *The quoting is compatible with UNIX shells*) will fail.